### DISCUSSION TITLE
[课程 7: 自定义硬件后端]你听懂了吗？

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-zh#99] [Polls]
个人感觉[这节课](https://www.bilibili.com/video/BV1QU4y1Y7q8/?spm_id_from=333.788&vd_source=c29c11f2ab83e7f4168f5f30ba5e2aca)比以往难度要大，对于所做的优化和生成的TVMScript一知半解，希望老师能多讲下细节。
如果大家有什么听课经验，也请分享一下，谢谢！

**Comment by kiloGrand** [comment]
ex = relax.vm.build(MLPModule, target="llvm")
想问一下老师，如果要build成安卓平台，target要怎么填，会不会有相关的课程项目呢？

**Reply by Hzfengsy** [reply]
可以参考[TVM文档](https://tvm.apache.org/docs/how_to/deploy_models/deploy_model_on_android.html)。课程安排方面可能会在最后一讲涉及ARM端部署（不一定是安卓），具体安排还没确定。

PS：独立问题可以考虑新开discussion :)
